yapapi/services.py

```diff
@@ -502,19 +524,17 @@ def payload(self) -> Payload:
         """Return the service runtime definition for this :class:`Cluster`."""
         return self._payload
 
-    @property
-    def service_class(self) -> Type[Service]:
-        """Return the class instantiated by all service instances in this :class:`Cluster`."""
-        return self._service_class
-
     @property
     def network(self) -> Optional[Network]:
         """Return the :class:`~yapapi.network.Network` record associated with the VPN used by this :class:`Cluster`."""
         return self._network
 
     def __repr__(self):
+        #   TODO: prettier print (cnt: service_class for every service_class)
+        service_classes = set(type(service).__name__ for service in self.instances)
+        service_classes = sorted(service_classes)
         return (
-            f"Cluster {self.id}: {len(self.__instances)}x[Service: {self._service_class.__name__}, "
+            f"Cluster {self.id}: {len(self.__instances)}x[Services: {service_classes}, "
```

Comment:
ugh.... this kinda proves that we don't want Clusters with multiple classes inside ... ;p

Reply:
"Pretty-print-based-desgn" :P?
There's a TODO in the function, this will be somehow addressed.